src/imagery/i.eodag/i.eodag.py

```diff
@@ -115,22 +130,19 @@ def create_dir(directory):
 
 
 def get_bb(vector=None):
```

Comment:
```suggestion
def get_bb():
```
Vector does not seem to be used in this function any more...